src/sentry/middleware/proxy.py

```diff
@@ -105,33 +30,3 @@ def process_request(self, request):
             real_ip = real_ip.split(",")[0].strip()
             real_ip = self._remove_port_number(real_ip)
             request.META["REMOTE_ADDR"] = real_ip
-
-
-class DecompressBodyMiddleware:
```

Comment:
We ran this by the ingest team yeah?

Reply:
Yes. Although, I found one thing that cropped up today for release files...

Reply:
Ok, this has been resolved.